src/components/OcDatepicker.vue

```diff
@@ -107,18 +118,48 @@ export default {
       type: [Object, String],
       default: null,
     },
+    /**
+     * Label of the datepicker.
+     **/
+    label: {
+      type: String,
+      required: true,
+      default: null,
+    },
+    /**
+     * A description text which is shown below the datepicker.
+     */
+    descriptionMessage: {
+      type: String,
+      default: null,
+    },
   },
   data() {
     return {
       party: true,
       update: this.date,
     }
   },
+  computed: {
+    messageId() {
+      return `${this.id}-message`
+    },
+    additionalAttributes() {
+      const additionalAttrs = {}
+      if (this.descriptionMessage) {
+        additionalAttrs["aria-describedby"] = this.messageId
+      }
+      return { ...this.$attrs, ...additionalAttrs }
+    },
+    ariaInvalid() {
```

Comment:
Not needed as of now. Please remove. :-) Same for the `aria-invalid` attribute in the template.